### PR TITLE
Convert to github actions and publish to NPM

### PR DESCRIPTION
**Pull request by EvanLemieux**
Setting this repo up to publish to NPM.

### CODE REVIEW

**Comment by marcsoiferman** [comment]
Should this publish to NPM or CodeArtifact?

**Comment by EvanLemieux** [comment]
> Should this publish to NPM or CodeArtifact?

This should publish to NPM, at least for now. This repo is used in a lot of places right now, and if I made this private (in order to publish to CodeArtifact) those other places would have to be made private as well.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
```
Comment (jameswklassen) [inline .github/workflows/release.yml:8]: `[skip ci]` is now a native [GitHub feature](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/) so I think this is no longer needed.

**Review by jameswklassen — APPROVED** [review]
One minor nit about a redundant conditional, but looks good otherwise 👍

**Pull request merged**